extensions-core/kafka-extraction-namespace/src/main/java/io/druid/query/extraction/namespace/KafkaExtractionNamespace.java

```diff
@@ -34,19 +35,19 @@
 {
   @JsonProperty
   private final String kafkaTopic;
+
   @JsonProperty
-  private final String namespace;
+  private final Properties kafkaProperties;
 
   @JsonCreator
   public KafkaExtractionNamespace(
       @NotNull @JsonProperty(value = "kafkaTopic", required = true) final String kafkaTopic,
-      @NotNull @JsonProperty(value = "namespace", required = true) final String namespace
+      @NotNull @JsonProperty(value = "kafkaProperites", required = true) final Properties kafkaProperties
```

Comment:
Does this need to be required? also required=true is just a hint, you still have to enforce not null checks in constructor to be ACTUALLY required.